hyperdome/common/common.py

```diff
@@ -42,13 +90,10 @@ class Common(object):
     def __init__(self, debug=False):
         self.debug = debug
 
-        # The platform OnionShare is running on
-        self.platform = platform.system()
-        if self.platform.endswith("BSD"):
-            self.platform = "BSD"
+        # The platform_str OnionShare is running on
```

Comment:
What's the comment for if the code is gone?
Also, Hyperdome*

Reply:
gotta leave something to do next PR.
I should really just find+replace all references to OnionShare and micahflee at this point, I don't think there's any files left untouched.